example/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
if I can remove that somehow i'd gladly take the option, I'll find a way if there's